extensions/warp-blink-wrtc/src/blink_impl/data/notify_wrapper.rs

```diff
@@ -0,0 +1,12 @@
+use std::sync::Arc;
+use tokio::sync::Notify;
+
+pub struct NotifyWrapper {
+    pub notify: Arc<Notify>,
+}
+
+impl Drop for NotifyWrapper {
+    fn drop(&mut self) {
+        self.notify.notify_waiters();
```

Comment:
you should probably check the strong count of the reference so if its the only thing left, it can signal its waiters. Eg

```rust
if Arc::strong_count(&self.notify) == 1 {
   self.notify.notify_waiters();
}
```

Reply:
`notify` gets cloned and passed to some tasks. I think that dropping NotifyWrapper` wouldn't necessarily mean the strong count would be one. I hoped that by putting NotifyWrapper in an Arc, Drop wouldn't be called until all the `Arc<NotifyWrapper>`s were dropped.